tests/functional/graphql/test_related.py

```diff
@@ -93,13 +94,107 @@ def publisher_json(item_id: str, pub_url: str):
+                        id
+                        corpusItem {
+                          id
+                        }
+                    }
+                }
+              }
+            }
+        ''',
+        'variables': {
+            'representations': [
+                {
+                    '__typename': 'Item',
+                    'itemId': item_id
+                },
+            ],
+        },
+    }
+
+
+def after_save_json(item_id: str):
+    return {
+        'query': '''
+            query ($representations: [_Any!]!) {
+              _entities(representations: $representations) {
+                ... on Item {
+                    itemId
+                    relatedAfterCreate(count: 3) {
+                        id
+                        corpusItem {
+                          id
+                        }
+                    }
+                }
+              }
+            }
+        ''',
+        'variables': {
+            'representations': [
+                {
+                    '__typename': 'Item',
+                    'itemId': item_id
+                },
+            ],
+        },
+    }
+
+
 class TestGraphQLRelated(TestCase):
     @classmethod
     def setUpClass(cls):
         test_data = populate_qdrant()
-        cls.art_by_corpus_id = {d['payload'] ['corpus_item_id']: d['payload'] for d in test_data}
+        cls.art_by_corpus_id = {d['payload']['corpus_item_id']: d['payload'] for d in test_data}
 
-    def test_related_end_of_article_basic(self):
+    def test_related_after_save(self):
+        """ recommend similar curated """
+        item_id = '3727699409'
+
+        with TestClient(app) as client:
+            response = client.post("/", json=after_save_json(item_id)).json()
+
+            assert not response.get('errors')
+            entity = response['data']['_entities'][0]
+            recs = entity['relatedAfterCreate']
+            assert entity['itemId'] == item_id
+            assert len(recs) == 3
+            assert 'id' in recs[0]
+            assert 'corpusItem' in recs[0]
+            assert 'id' in recs[0]['corpusItem']
+            print(recs)
+            assert all(self.art_by_corpus_id[r['corpusItem']['id']]['is_curated'] for r in recs)
+
+    def test_related_after_article(self):
+        """ recommend similar curated """
+        item_id = '3727699409'
+
+        with TestClient(app) as client:
+            response = client.post("/", json=after_article_json(item_id)).json()
+
+            assert not response.get('errors')
+            entity = response['data']['_entities'][0]
+            recs = entity['relatedAfterArticle']
+            assert entity['itemId'] == item_id
+            assert len(recs) == 3
+            assert 'id' in recs[0]
+            assert 'corpusItem' in recs[0]
+            assert 'id' in recs[0]['corpusItem']
+            print(recs)
+            assert all(self.art_by_corpus_id[r['corpusItem']['id']]['is_curated'] for r in recs)
```

Comment:
Is the `print` just debug code? If helpful you can optionally add a message to the end of an assertion.
```suggestion
            assert all(self.art_by_corpus_id[r['corpusItem']['id']]['is_curated'] for r in recs), recs
```